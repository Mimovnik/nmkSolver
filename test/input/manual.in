GEN_ALL_POS_MOV 3 2 3 1
1 0
2 2
2 1
SOLVE_GAME_STATE 3 3 3 2
1 0 0
0 0 0
0 0 0
SOLVE_GAME_STATE 3 3 3 1
1 2 0
0 0 0
0 0 0