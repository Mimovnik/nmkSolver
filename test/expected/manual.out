1
1 1
2 2
2 1
BOTH_PLAYERS_TIE
FIRST_PLAYER_WINS
SECOND_PLAYER_WINS
